frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts

```diff
@@ -15,6 +14,7 @@ import {debounceTime, skip, take} from "rxjs/internal/operators";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {Observable} from "rxjs";
 import {BoardFiltersService} from "core-app/modules/boards/board/board-filter/board-filters.service";
+import {APIV3Service} from "core-app/modules/apiv3/api-v3.service";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/e86108ab2c62ce6134c40a6d3c952e50791b7254/frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts#L18"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>